packages/terra-notification-dialog/src/NotificationDialog.jsx

```diff
@@ -112,179 +103,119 @@ const propTypes = {
-            <div className={cx('notification-dialog-container')} tabIndex="0">
-              <div id="notification-dialog-header" className={cx('header-body')}>{header || defaultHeader}</div>
-              <div className={cx('notification-dialog-body')}>
-                {variant
-                  && <div className={cx('icon-container')}>{getIcon(variant, customIcon)}</div>}
-                <div className={cx('text-wrapper')}>
-                  {title
-                    && <div id="notification-dialog-title" className={cx('title')}>{title}</div>}
-                  {(startMessage || message)
-                    && <div className={cx('message')}>{(startMessage || message)}</div>}
-                  {content
-                    && <div>{content}</div>}
-                  {endMessage
-                    && <div className={cx('message')}>{endMessage}</div>}
+  /* eslint-disable jsx-a11y/no-noninteractive-tabindex */
+  return (
+    <AbstractModal
+      ariaLabel="Notification Dialog"
+      aria-labelledby="notification-dialog-header"
+      aria-describedby={title ? 'notification-dialog-title' : 'notification-dialog-header'}
+      role="alertdialog"
+      classNameModal={classNames(cx('notification-dialog', theme.className), customProps.className)}
+      isOpen={isOpen}
+      onRequestClose={() => {}}
+      closeOnEsc={false}
+      closeOnOutsideClick={false}
+      zIndex="9000"
+      data-terra-notification-dialog
+    >
+      <FocusTrap focusTrapOptions={{ returnFocusOnDeactivate: true, clickOutsideDeactivates: false, escapeDeactivates: false }}>
+        <div className={cx('notification-dialog-inner-wrapper')}>
+          <div className={cx('notification-dialog-container')} tabIndex="0">
+            <div className={cx(['floating-header-background', variant])} />
+            <div className={cx(['header'])}>
+              <div className={cx(['header-content'])}>
+                <NotificationIcon variant={variant} iconClassName={(custom || {}).iconClassName} />
+                <div className={cx('header-container')}>
+                  <div id="notification-dialog-signal-word" className={cx('signal-word')}>{signalWord}</div>
+                  <div id="notification-dialog-title" className={cx('title')}>{title}</div>
                 </div>
               </div>
-              <div className={cx('footer-body')}>
-                {actionSection(
-                  acceptAction || primaryAction,
-                  rejectAction || secondaryAction,
-                  buttonOrder,
-                  emphasizedAction,
-                )}
-              </div>
+            </div>
+            <div className={cx('body')}>
+              {(startMessage)
+                && <div className={cx('message')}>{(startMessage)}</div>}
+              {content
+                && <div className={cx('message')}>{content}</div>}
+              {endMessage
+                && <div className={cx('message')}>{endMessage}</div>}
```

Comment:
These three divs share the same `message` className, is this correct? Specifically for the content region.

Reply:
I think this is correct to align with the zeplin callouts for applying the padding / font-size / color since content could be anything. I will have to double check & get back.

Reply:
My concern here would be that `content` accepts a node type, should we apply styles to the wrapper of custom content? It might make sense depending on what our expectations for content is.

Reply:
This is an okay assumption to provide standard starter styling so that basic content (text size, color, vertical spacing between the other messages/text) correctly matches the styles of the notification dialog. The intent of a node prop type here _is not_ to provide consumers endless customization of inner content. The intent is to provide small utility to include lists, hyperlinks, and the occasional show/hide (include guid or tracing error code). 

For anything beyond basic text, Consumers should be expected to provide any additional spacers, style, or resets for elements they are adding - based on their needed use case and working with their UX designers.